src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/WorkspaceModelEntities.kt

```diff
@@ -53,7 +53,9 @@ public data class Library(
   val iJars: List<String> = listOf(),
   val sourceJars: List<String> = listOf(),
   val classJars: List<String> = listOf(),
-) : WorkspaceModelEntity(), ResourceRootEntity {
+  val goImportPath: String? = "",
+  val goRoot: URI? = URI(""),
```

Comment:
Dla mnie to sprawi, że kod będzie znacznie ładniejszy:
Jeśli z `ModuleDetails` usuniemy `goLibraries`, to okazuje się, że `goImportPath` i `goRoot` z `Library` i `LibraryItem` są używane jedynie przez `GoModule`, a tam z `LibraryItem` zawsze korzystamy tylko z tych dodatkowych pól dla GO

To sprowadza się do tego, ze możemy zrobić cos w stylu `GoLibrary`, które będzie miało tylko te 2 pola od GO i wtedy w `GoModule` `goLibraries` możemy zdefiniować jako listę `GoLibrary` - w wielu miejscach to znacznie uprości i zachowuje rozdzielenie domenowe między językami